portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
```

Comment:
This will soon be irrelevant (because you'll be changing this to something else), but this `setContentType()` parameter could be corrected, aligning your change of the response message string (dropping `<h1>` tag). This doesn't affect the outcome of this task, and you'll be changing this in the next step anyway, so no need to correct this for step 2 only.